pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/OncHistoryDetail.java

```diff
@@ -297,6 +297,7 @@ public OncHistoryDetail(Integer oncHistoryDetailId, Integer medicalRecordId, Str
         this.participantId = participantId;
         this.ddpParticipantId = ddpParticipantId;
         this.ddpInstanceId = ddpInstanceId;
+        this.deleted =  (deleted != null && deleted == 1);
```

Comment:
It would be great to backfill `deleted` and make sure it's always non-null so that we can use `int` (or better yet, `boolean`) and avoid null checks like this.

We could also pull out a `Integer.valueOf(1)` into a constant like `INTEGER_TRUE` and then simplify this with `INTEGER_TRUE.equals(deleted)`